docs/dynamic-tao/staking-unstaking-dtao.md

```diff
@@ -1,47 +1,8 @@
 ---
-title: "Staking and Unstaking in Dynamic TAO"
+title: "Understanding pricing and anticipating slippage"
 ---
-import ThemedImage from '@theme/ThemedImage';
-import useBaseUrl from '@docusaurus/useBaseUrl';
 
-# Staking and Unstaking in Dynamic TAO
-
-This page refers to the nature of staking and unstaking operations in the [Dynamic TAO model](./index.md), which is a planned evoluation of the Bittensor network.
-
-Staking is always local to a subnet.
-
-Each subnet operates its own automated market-maker (AMM), meaning it mantains its own reserves of the two tokens being traded so that it can facilitate a trade of any desired quantity of liquidity (as long as its available), at a price that it automated calculates.
-
-Each subnet has a reserve of TAO and a reserve of its currency, referred to in general as its alpha ($\alpha$) currency. Stake is held in $\alpha$ token denominations.
-
-As a TAO holder you will stake to a validator’s hotkey (as previously), but now you select a subnet to stake to them within.
-
-
-**When you stake:**
-
-1. First, your TAO stake goes into the subnet's TAO reserve of its AMM (automated market maker) pool.
-1. Then, the subnet AMM pool algorithm uses the exchange rate and calculates the equivalent units of $\alpha$, for the TAO that was just added to the TAO reserve side. This amount of $\alpha$ is taken out of the alpha reserve of the pool and is sent to the validator’s hotkey. 
-1. The validator’s hotkey holds the $\alpha$. The sum of stake among all hotkeys is referred as **$\alpha$ outstanding** for that subnet. 
-
-**When you unstake:**
-
-1. When you issue an unstake command, `btcli stake remove`, and specify the units of $\alpha$ token you want to unstake, this $\alpha$ is first taken out of the validator’s hotkey and added to the $\alpha$ reserves of the subnet pool. 
-2. The subnet AMM pool algorithm then applies the latest exchange rate and calculates the equivalent TAO units for the $\alpha$ token units that were just added to the $\alpha$ reserves of the pool. 
-3. These equivalent TAO units are then taken out of the TAO reserves of the subnet pool and are sent to the TAO holder’s coldkey.
-
-:::tip Stake is always expressed in alpha units
-In Dynamic TAO, except for the stake held in [the Root Subnet](#root-subnet-subnet-zero), the stake held by a hotkey in a subnet is always expressed in the subnet-specific $\alpha$ units. Root Subnet stake is expressed in $\tau$.
-:::
-
-:::tip Prereq
-To follow along, install the Dynamic TAO-enabled release candidate of the Bittensor command line interface `btcli`, by running:
-```shell
-pip install bittensor-cli==8.2.0rc10
-```
-:::
-
-
-## Understanding pricing and anticipating slippage
+# Understanding pricing and anticipating slippage
```

Comment:
"Understanding Pricing and Anticipating Slippage" to keep it consistent with the other docs